backend/kitschy_api/tests/product_image_tests.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from model_bakery import baker
+
+"""
+Tests:
+1. TestProductImageAPI - CRUD operations
+2. Edge Cases - Empty data
+"""
+
+
+@pytest.mark.django_db
+class TestProductImageAPI:
+    endpoint = "/api/products-image/"
+
+    def test_post_product_image(self, authenticated_user):
+        product = baker.make("Product")
+        product_image_data = {
+            "product": product.product_id,
+            "img_url": "http://image.com",
+            "alt_desc": "An image of the product",
+        }
+        response = authenticated_user.post(self.endpoint, product_image_data)
+
+        assert response.status_code == 201
+        assert response.data["product"] == product.product_id
+
+    def test_update_product_image(self, authenticated_user):
+        product_image = baker.make("ProductImage")
+        response = authenticated_user.put(
```

Comment:
We use the `PATCH` request for updating, so that we only have to provide the keys to update.